### PR TITLE
Improve style of the IdentityCard expanded view

### PR DESCRIPTION
**Pull request by evamillan**
This PR moves the identities and organizations data in the compact view of `ExpandedIndividual` to Vuetify simple tables to
improve readability. The data sources are replaced with their icon to save space.

This is an example of the improved view:
imagen

### CODE REVIEW

File: ui/.storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
```
Comment (sduenas) [inline ui/.storybook/preview-head.html:2]: Can we change this to static fonts? There are some concerns about GDPR and Google Fonts.

https://complianz.io/google-fonts-and-gdpr-does-it-work/

File: ui/src/components/ExpandedIndividual.vue [hunk 2]
```diff
@@ -48,24 +48,45 @@
...
+            <th class="text-left">Username</th>
+            <th class="text-left">Source</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="item in flatIdentities" :key="item.uuid">
+            <td>{{ item.name || "-" }}</td>
+            <td>{{ item.email || "-" }}</td>
+            <td>{{ item.username || "-" }}</td>
+            <td><v-icon v-text="item.icon" small left /></td>
```
Comment (sduenas) [inline ui/src/components/ExpandedIndividual.vue:66]: Is it possible to add the name of the source when the cursor is over the icon?

**Review by sduenas — CHANGES_REQUESTED** [review]
Looks great but I have some comments.

**Comment by evamillan** [comment]
@sduenas I added the fonts to the project assets, the app was also loading them from Google Fonts. I also added the name of the source to the icons.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**